Update the description of `update-all-matching-relationships` (#2105)
neo4j/docs-operations: modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

|false

|--update-all-matching-relationships

> |label:new[Introduced in 2025.01] If one relationship data entry matches multiple existing relationships, this decides whether to update all matching or to log as an error.

Reviewer: How about this:
```suggestion
|label:new[Introduced in 2025.01] Updates all existing relationships that match a relationship data entry.
When disabled, it throws an error.
```
or
```
Whether or not to update all existing relationships that match a relationship data entry.
```
